node/sc/bridge_manager_test.go

```diff
@@ -201,42 +207,10 @@ func TestBridgeManager(t *testing.T) {
 				assert.NoError(t, err)
 				assert.Equal(t, false, done)
 
-				switch ev.TokenType {
-				case KLAY:
-					tx, err := bridge.HandleKLAYTransfer(
-						&bind.TransactOpts{From: cAuth.From, Signer: cAuth.Signer, GasLimit: testGasLimit},
-						ev.Raw.TxHash, ev.From, ev.To, ev.ValueOrTokenId, ev.RequestNonce, ev.Raw.BlockNumber, ev.ExtraData)
-					if err != nil {
-						log.Fatalf("Failed to HandleKLAYTransfer: %v", err)
-					}
-					fmt.Println("WithdrawKLAY Transaction by event ", tx.Hash().Hex())
-					sim.Commit() // block
-
-				case ERC20:
-					tx, err := bridge.HandleERC20Transfer(
-						&bind.TransactOpts{From: cAuth.From, Signer: cAuth.Signer, GasLimit: testGasLimit},
-						ev.Raw.TxHash, ev.From, ev.To, tokenAddr, ev.ValueOrTokenId, ev.RequestNonce, ev.Raw.BlockNumber, ev.ExtraData)
-					if err != nil {
-						log.Fatalf("Failed to HandleERC20Transfer: %v", err)
-					}
-					fmt.Println("HandleERC20Transfer Transaction by event ", tx.Hash().Hex())
-					sim.Commit() // block
-
-				case ERC721:
-					owner, err := nft.OwnerOf(&bind.CallOpts{From: pAuth.From}, big.NewInt(int64(nftTokenID)))
-					assert.Equal(t, nil, err)
-					fmt.Println("NFT owner before HandleERC721Transfer: ", owner.String())
-
-					tx, err := bridge.HandleERC721Transfer(
-						&bind.TransactOpts{From: cAuth.From, Signer: cAuth.Signer, GasLimit: testGasLimit},
-						ev.Raw.TxHash, ev.From, ev.To, nftAddr, ev.ValueOrTokenId, ev.RequestNonce, ev.Raw.BlockNumber, ev.Uri, ev.ExtraData)
-					if err != nil {
-						log.Fatalf("Failed to HandleERC721Transfer: %v", err)
-					}
-					fmt.Println("HandleERC721Transfer Transaction by event ", tx.Hash().Hex())
-					sim.Commit() // block
-				}
+				bridgeInfo.AddRequestValueTransferEvents([]*RequestValueTransferEvent{ev})
+				bridgeInfo.processingPendingRequestEvents()
```

Comment:
Could you please explain this code change with detail?

Reply:
I added some comments. :)
1st routine is to insert the event to bridgeInfo's event list.
2nd routine is to handle the event in the event list.